CONTRIBUTING.md

```diff
@@ -230,3 +231,11 @@ git commit -s -S -m "Add a meaningful commit message"
 git push origin issue-number/contribution-description
 ```
 6. Open a pull request from your forked repository branch to the main branch of the 'galasa repository', and explain your changes. Refer to any stories which are relevent and explain why the change was made, what the change is, and anything else which reviewers would find helpful to understand the context of the change.
+
+## How to make a contribution to Galasa's webui
+
+1. Similarly to the main Galasa repository, commits [must be signed](#sign-your-commits).
+
+2. If you do not have access to your own Galasa service, either production or Minikube, development will be aided by access to one. To get access please [sign up](https://github.com/orgs/galasa-dev/teams/galasa-service1-users) to be a part of the galasa-service1-users team on Github. On that page there should be a request to join button, which will send a request to one of the maintainers, so please reach out on one of the us otherwise your request will be denied. For example, this could be done in the comments of a ticket you would like to do on [our Kanban](https://github.com/orgs/galasa-dev/projects/3). Next, go to https://galasa-service1.galasa.dev/ which will get you to authenticate via Dex, then you'll be added to the system with the 'viewer' role. 
```

Comment:
A few suggestions:

- "If you do not have access to your own Galasa service, either production or Minikube, development will be aided by access to one" could be clarified to something like "Having access to a Galasa service can help aid development of the web UI by removing the need to set up additional back-end services locally"
- "reach out on one of the us" doesn't sound quite right! Maybe "reach out to the Galasa team via our support channels (see https://galasa.dev/support)"
- "of a ticket you would like to do on our Kanban" could be clarified to "of a ticket you would like to work on from our Kanban board"
- "authenticate via Dex" should be "authenticate via GitHub"